demo/src/App.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from "react";
```

Comment:
🛑 Should be `import * as React`

Reply:
`create-react-app` has it's own style... updating to match ours. :+1:

Reply:
Ah tricky, ok!

Reply:
🛑 Oh - and we should have `// @flow` on this file, too!